src/cat/controls/initFileLoad.js

```diff
@@ -0,0 +1,84 @@
+export function initFileLoad() {
+  var cat = this;
+  //draw the control
+  var loadLabel = cat.controls.dataWrap.append("p").style("margin", 0);
+
+  loadLabel
+    .append("small")
+    .text("Use local .csv file:")
+    .append("sup")
+    .html("&#9432;")
+    .property(
+      "title",
+      "Render a chart using a local file. File is added to the data set list, and is only available for a single session and is not saved."
+    )
+    .style("cursor", "help");
+
+  var loadStatus = loadLabel
+    .append("small")
+    .attr("class", "loadStatus")
+    .style("float", "right")
+    .text("Select a csv to load");
+
+  cat.controls.dataFileLoad = cat.controls.dataWrap
+    .append("input")
+    .attr("type", "file")
+    .attr("class", "file-load-input")
+    .on("change", function() {
+      if (this.value.slice(-4) == ".csv") {
```

Comment:
Maybe `toLowerCase()` this just to cover all the bases.

Reply:
Fixed in `hotfixes` branch.